modules/fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/fs2/config/io.scala

```diff
@@ -98,4 +100,26 @@ object io {
     implicit val outputEncoder: Encoder[Output] =
       deriveConfiguredEncoder[Output]
   }
+
+  sealed trait MetricsReporter {
+    def period: FiniteDuration
+  }
+
+  object MetricsReporter {
+    case class Stdout(period: FiniteDuration) extends MetricsReporter
+    case class StatsD(
+      period: FiniteDuration,
+      hostname: String,
+      port: Int,
+      tags: Map[String, String]
+    ) extends MetricsReporter
+
+    implicit val finiteDurationEncoder: Encoder[FiniteDuration] =
+      implicitly[Encoder[String]].contramap(_.toString)
+
+    implicit val metricsReporterDecoder: Decoder[MetricsReporter] =
+      deriveConfiguredDecoder[MetricsReporter]
```

Comment:
I thought we were going to add a fallback option with `metricsReportPeriod` to make configuration foward-compatible?